scripts/schema-cmp.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+# usage: schema-cmp.py expected.json actual.json
+# schema comparison should match, except timezones which we just spot
+# check for a few expected values and expect that reality is a superset
+import json
+import sys
+
+def load(filename):
+    with open(filename, 'r') as f:
+        data = json.load(f)
+    tz = data['properties']['timezone']['enum']
+    del data['properties']['timezone']['enum']
```

Comment:
you can write this as tz = data['properties']['timezone'].pop('enum') if you like